code/modules/uplink/uplink_items.dm

```diff
@@ -22,6 +22,8 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 			continue
 		if (I.restricted && !allow_restricted)
 			continue
+		if (I.restricted_loneop && !allow_restricted_loneop)
+			continue
```

Comment:
There should be a better way of disabling items from loneops without adding a whole new flag.

Reply:
Not easily, that I can think of anyway. Game mode is set to nuclear whenever an nukie uplink initializes and a lot of the other stuff rely on that, so not even considering the false positives it is already useless. Checking role would render uplinks unusable to anyone but the antags, and also require flags or something to track what is shown to who. Would also cause issues with admin spawned uplinks and improperly spawned antags. 


But if you got suggestions, I am listening.

Reply:
most of those stupid vars should be a single bitflag